rollup.config.js

```diff
@@ -26,14 +25,7 @@ export default function rollup() {
         exclude: /node_modules/,
         extensions: [".ts"]
       }),
-      nodeResolve({ extensions: [".ts"] }),
-      copy({
```

Comment:
this copy is redundant bc these three files get published by default, and the inclusion of which messes with publishing from the root folder.

If the goal is to publish from the `build` directory, then I could see adding it back